doc/build.md

```diff
@@ -97,6 +97,14 @@ b) [RISCV GNU](https://github.com/riscv-collab/riscv-gnu-toolchain)
 c) [RISCV64 GCC](https://packages.ubuntu.com/bionic/gcc-riscv64-linux-gnu) for RISCV64 only
   - `sudo apt-get install gcc-riscv64-linux-gnu`
 
+d) [RISCV NONE](https://gcc.gnu.org/)
+  - Use a GCC compiler configured for building baremetal (not Linux) binaries
+
+  - For example on Arch you can install it with
+      ```
+      sudo pacman -Syu riscv32-elf-binutils riscv32-elf-newlib riscv64-elf-binutils riscv64-elf-gcc riscv64-elf-newlib

```

Comment:
what is command for ubuntu? I got failure below:

```
jyao1@jyao1-desk:~$ sudo apt-get install riscv32-elf-binutils riscv32-elf-newlib riscv64-elf-binutils riscv64-elf-gcc riscv64-elf-newlib
[sudo] password for jyao1:
Reading package lists... Done
Building dependency tree
Reading state information... Done
E: Unable to locate package riscv32-elf-binutils
E: Unable to locate package riscv32-elf-newlib
E: Unable to locate package riscv64-elf-binutils
E: Unable to locate package riscv64-elf-gcc
E: Unable to locate package riscv64-elf-newlib
```

Reply:
I don't use Ubuntu, so I'm not sure.

I have written the instructions to be clear for the distro that I use, in the same way the rest of the instructions are only clear for Ubunutu users and not other users. I hopefully have provided enough information to explain what is required so that people can figure it out for their distro

Reply:
It is OK if you only validated Arch Linux.

My request is that: **you should say: it is only validated in Arch Linux.**

If so, the right URL should be  https://archlinux.org/packages/extra/x86_64/riscv64-elf-gcc/

Reply:
I have updated the instructions to more explicitly say I have only tested this on Arch

Reply:
Thank you!